### DISCUSSION TITLE
Make it possible to disable Let's Encrypt SSL provider on per service basis.

### DISCUSSION THREAD

**Discussion by nuxwin** [coollabsio/coolify#3581] [Feature Requests]
Good morning,

When adding a resource (i.e. Docker compose), and if the domain is provided with the `https//` scheme, Coolify automatically add traefik labels for both TLS and Let's Encrypt certresolver. It could be great to add a checkbox, on per service basis, to enable (default), or disable the Let's Encrypt certresolver.

Why ?

For one of our service, the target server is behind a proxy (hosted on another server) which handle SSL termination (with a valid SSL certificate).

On the target server, only port 443 is exposed to the WAN. The target server make use of a self-signed SSL cert which is a default SSL certificate.

Basically put : 

WAN -> Server (proxy with valid SSL certificate) -> target server (port 443) (traefik with self-signed SSL cert) -> backend (container port 5000, not exposed to the host)

We added our self-signed SSL certificate with dynamic conffile. This seem to work but we want disable Let's Encrypt explicitely.

![Capture d’écran 2024-09-26 à 12 50 26](https://github.com/user-attachments/assets/7beb7571-fc74-4130-a4e1-0ca63394695c)
![Capture d’écran 2024-09-26 à 12 51 03](https://github.com/user-attachments/assets/19e01ac1-cd54-4212-afb7-34a9e0c1855a)

**Comment by kolezka** [comment]
Bump!

**Comment by kevineduardo** [comment]
I need that too!

**Comment by mredodos** [comment]
+1

**Comment by D3CK3R** [comment]
+1

**Comment by mikkaK** [comment]
+1

**Comment by Rustypredator** [comment]
If switching to Caddy proxy is acceptable for anyone with this problem, see my discussion: https://github.com/coollabsio/coolify/discussions/4726

**Comment by alexgorbatchev** [comment]
+1 to this, in a cloudflare -> coolify setup coolify proxy doesnt need SSL

**Comment by Zerebel** [comment]
+1

**Comment by dxqwww** [comment]
+1

**Comment by ztane** [comment]
I have a different problem which *still* could be squashed with the same solution: I need to use different resolvers - dns and http acme, depending on domains. There should be support for completely *unsetting* and *changing* the value of the resolver field per container basis.

**Comment by teofeo** [comment]
+1

**Comment by muncherelli** [comment]
+1
